src/lib/constants.ts

```diff
@@ -68,6 +68,9 @@ const constants = {
 	supervisorNetworkGateway: '10.114.104.1',
 	// How often can we report our state to the server in ms
 	maxReportFrequency: 10 * 1000,
+	// How often can we report metrics to the server in ms; mirrors server setting.
+	// Metrics are low priority, so less frequent than maxReportFrequency.
+	maxMetricsFrequency: 300 * 1000,
```

Comment:
I would move this definition to the top of `report.ts`. If we ever need the value somewhere else we can move it. I think constants.ts is bloated enough

Reply:
Move `maxReportFrequency` as well? It also is used only in `report.ts`.

Reply:
Yep, sounds good to me.